### DISCUSSION TITLE
Update your documentation!!

### DISCUSSION THREAD

**Discussion by wisquimas** [containrrr/watchtower#1397] [General] [answered]
![image](https://user-images.githubusercontent.com/5402171/192655225-5d315a64-f9a5-4f1b-b118-66429927f6f6.png)

In this photo you have 
      - <PATH_TO_HOME_DIR>/.docker/config.json:/config.json
But is:
      - <PATH_TO_HOME_DIR>/.docker/config.json:/config/config.json

Two hours with this......

**Comment by piksel** [comment, marked as answer]
If you have not overridden `DOCKER_CONFIG` with any other value, the default config location is `/`.

**Reply by piksel** [reply]
Perhaps you copied the example overriding it in the [following section](https://containrrr.dev/watchtower/private-registries/#docker_config_path)?

**Comment by simonaronsson** [comment]
> Two hours with this......

This is an open-source project and everyone who contributes to the project is doing it pro bono and because they think it is worthwhile pouring time into. This also means that docs sometimes might get outdated, or take longer than ideal to complete. However, this does not even seem to have been the case this time, as @piksel already pointed out.

Please leave your sense of entitlement at the door.
